tests/unit/php/Core/Content/Media/Pathname/PathnameStrategyTest.php

```diff
@@ -1,6 +1,6 @@
 <?php declare(strict_types=1);
 
-namespace src\Core\Content\Test\Media\Pathname;
```

Comment:
Hey, this test needs to be moved to the other integration tests. Also, you need to provide a unit test for this. For more information, contact @shyim

Reply:
I don't know, why I should add a unit for the named test, because I just moved it :-) It's nothing introduced by me, but moved it now to the integrations folder.

working on the PHPstan issues in my touched files 👍

Reply:
@NiklasLimberg ready! :-)

Reply:
It does not really matter, it will be squashed when it's merged